extensions/amp-video-docking/0.1/amp-video-docking.js

```diff
@@ -936,16 +784,14 @@ export class VideoDocking {
   }
 
   /**
-   * @param {!../../../src/video-interface.VideoOrBaseElementDef} video
-   * @param {!DockTargetDef} target
+   * @param {!VideoOrBaseElementDef} video
+   * @param {!TargetDef} target
```

Comment:
I think `DockTargetDef` is less ambiguous here, any reason why we're dropping the `Dock`?